XLocalizer/Xml/XmlResourceProvider.cs

```diff
@@ -38,7 +38,7 @@ private string ResourcePath<TResource>()
             {
                 string typeName = ResourceTypeHelper.CreateResourceName(typeof(TResource), _options.ResourcesPath);
 
-                return $".\\{_options.ResourcesPath}\\{typeName}.{{0}}.xml";
+                return $"{_options.ResourcesPath}/{typeName}.{{0}}.xml";
```

Comment:
Instead of hard coding the directory separator char `/`, we can use [`Path.DirectorySeparatorChar`](https://docs.microsoft.com/en-us/dotnet/api/system.io.path.directoryseparatorchar?redirectedfrom=MSDN&view=net-5.0) as below:

````cs
return $"{_options.ResourcesPath}{Path.DirectorySeparatorChar}{typeName}.{{0}}.xml";
````

Reply:
Yes. Fully agree